app/models/user.rb

```diff
@@ -9,4 +9,8 @@ class User < ApplicationRecord
   }
 
   attribute :role, :string, default: "delivery_partner"
+
+  def role_name
```

Comment:
This should probably use i18n instead, but [enums are weird](https://www.kostolansky.sk/posts/localize-rails-enums/) and don't support it out of the box.

Reply:
Okay cool, how about:

```ruby
def role_name
  t("activerecord.attributes.user.roles.#{role}")
end
````